src/tabs/presets/TitlePanel/PresetTitlePanel.css

```diff
@@ -20,7 +20,7 @@
     overflow: hidden;
     white-space: nowrap;
     text-overflow: ellipsis;
-    width: calc(100% - 30px);
+    width: calc(100% - 60px);
```

Comment:
@sugaarK this (I guess) trims the length of the title.

But agree we should set a limit in the preset itself.

Reply:
Anyway this should be ready to go